libyul/optimiser/StackLimitEvader.cpp

```diff
@@ -66,12 +67,37 @@ struct MemoryOffsetAllocator
 		if (unreachableVariables.count(_function))
 		{
 			yulAssert(!slotAllocations.count(_function), "");
+
+			if (functionDefinitions.count(_function))
```

Comment:
Perhaps
```suggestion
			if (_function != YulString{})
```

This is the only case where `_function` is not in `functionDefinitions`, right?

Reply:
IIRC the call graph contains the names of builtins, which won't be in ``functionDefinitions``.

Reply:
Actually, those won't end up in ``unreachableVariables```, though...

Reply:
But well. At the time of writing we also didn't have ``util::valueOr*`` yet, I'll just use that.